docs/extensions/engines/spark/lineage.md

```diff
@@ -207,3 +208,24 @@ spark.sparkContext.addSparkListener(new SparkListener {
 #### Get Lineage Events from Kyuubi EventHandler
 
 When using the `KYUUBI_EVENT` dispatcher, the lineage events will be sent to the Kyuubi `EventBus`. Refer to [Kyuubi Event Handler](../../server/events) to handle kyuubi events.
+
+#### Ingest Lineage Entities to Apache Atlas
+
+The lineage entities can be ingested into [Apache Atlas](https://atlas.apache.org/) using the `ATLAS` dispatcher.
+
+Extra works:
+
++ The least transitive dependencies needed, which are under `./extensions/spark/kyuubi-spark-lineage/target/scala-${scala.binary.version}/jars`
++ Use `spark.files` to specify the `atlas-application.properties` configuration file for Atlas
+
+Atlas Client configurations (Configure in `atlas-application.properties` or passed in `spark.atlas.` prefix):
```

Comment:
those sentences are not consistent, BTW, does it work to put `atlas-application.properties` under `SPARK_CONF_DIR`?

Reply:
Not sure, I'm afraid not so?
1. `org.apache.atlas.ApplicationProperties#get` will load atlas-application.properties from class path, ( and `atlas.conf` path will be preferred if the system env `atlas.conf` set )
2. `AtlasClientConf#clientConf` in plugin loads spark's configs with `spark.atlas.` prefix and overrides

Reply:
The `atlas-application.properties` file is required, I just provide the `spark.atlas.` prefix to supplement it